lib/mongo/cached_cursor.rb

```diff
@@ -0,0 +1,60 @@
+# A Cursor that attempts to load documents from memory first before hitting
+# the database if the same query has already been executed.
+#
+# @since 5.0.0
```

Comment:
Can you move this between lines 6 and 7? YARD won't understand that this comment is related to the `CachedCursor` class unless it's directly above the class definition.